addons/mrp/models/mrp_production.py

```diff
@@ -202,6 +203,21 @@ def _get_default_location_dest_id(self):
     production_location_id = fields.Many2one('stock.location', "Production Location", related='product_id.property_stock_production', readonly=False)
     picking_ids = fields.Many2many('stock.picking', compute='_compute_picking_ids', string='Picking associated to this manufacturing order')
     delivery_count = fields.Integer(string='Delivery Orders', compute='_compute_picking_ids')
+    confirm_cancel = fields.Boolean(compute='_compute_confirm_cancel')
+
+    @api.depends('move_raw_ids.state', 'move_finished_ids.state')
+    def _compute_confirm_cancel(self):
+        domain = [
+            ('production_id', 'in', self.ids),
```

Comment:
`` python3
domain = [
   ('state', '=', 'done'),
    '|',
        ('production_id', 'in', self.ids),
        ('raw_production_id', 'in', self.ids)
]
``